icu4c/source/i18n/complexunitsconverter.h

```diff
@@ -31,26 +32,27 @@ class U_I18N_API ComplexUnitsConverter {
      * NOTE:
      *   - inputUnit and outputUnits must be under the same category
      *      - e.g. meter to feet and inches --> all of them are length units.
-     * @param inputUnit represents the source unit. (should be single unit)
-     * @param outputUnits a single unit or multi units. For example (`inch` or `foot+inch`)
-     * @param lengthOfOutputUnits represents the length of the output units.
+     *
+     * @param inputUnit represents the source unit. (should be single or compound unit).
+     * @param outputUnits represents the output unit. could be any type. (single, compound or mixed).
      * @param status
      */
-    ComplexUnitsConverter(const MeasureUnit inputUnit, const MeasureUnit outputUnits,
+    ComplexUnitsConverter(const MeasureUnit &inputUnit, const MeasureUnit &outputUnits,
                           const ConversionRates &ratesInfo, UErrorCode &status);
 
-    // Returns true if the `quantity` in the `inputUnit` is greater than or equal than the `limit` in the
-    // biggest `outputUnits`
+    // Returns true if the specified `quantity` of the `inputUnit`, expressed in terms of the biggest
+    // unit in the MeasureUnit `outputUnit`, is greater than or equal to `limit`.
     //    For example, if the input unit is `meter` and the target unit is `foot+inch`. Therefore, this
     //    function will convert the `quantity` from `meter` to `foot`, then, it will compare the value in
     //    `foot` with the `limit`.
     UBool greaterThanOrEqual(double quantity, double limit) const;
 
-    // Returns outputMeasures which is an array with the correspinding values.
+    // Returns outputMeasures which is an array with the corresponding values.
     //    - E.g. converting meters to feet and inches.
     //                  1 meter --> 3 feet, 3.3701 inches
     //         NOTE:
-    //           the smallest element is the only element that could has fractional values.
+    //           the smallest element is the only element that could have fractional values. And all
+    //           other elements are rounded to the nearest integer
```

Comment:
Actually, more correct than saying "rounded" would be "floored".

Reply:
Done!